dataprofiler/data_readers/data_utils.py

```diff
@@ -553,12 +555,21 @@ def load_as_str_from_file(file_path, file_encoding, max_lines=10,
             
             # Return either the last index of sample_lines OR
             # the index of the newline char that matches remaining_lines
+            search_query_value = '\n'
+            if (isinstance(sample_lines, bytes)):
+                search_query_value = b'\r\n'
```

Comment:
i'm concerned this might not always be true that a byte file would have a \r\n and instead just have one or the other.